src/state/lottery/getUserLotteryData.ts

```diff
@@ -111,31 +113,13 @@ const getGraphLotteryUser = async (account: string): Promise<LotteryUserGraphEnt
 
 const getUserLotteryData = async (account: string, currentLotteryId: string): Promise<LotteryUserGraphEntity> => {
   const idsForTicketsNodeCall = getRoundIdsArray(currentLotteryId)
-  const ticketsToRequestPerRound = '5000'
```

Comment:
Deletions are are because we're able to massively simplify the amount of data processing done when not using multicall.